dsl_parser/interfaces/utils.py

```diff
@@ -72,7 +74,7 @@ def merge_schema_and_instance_inputs(schema_inputs,
         flattened_schema_inputs.items() +
         instance_inputs.items())
 
-    validate_missing_inputs(merged_inputs)
+    validate_missing_inputs(merged_inputs, schema_inputs)
```

Comment:
The name of the argument here is `schema_inputs`, but name of the parameter of `validate_missing_inputs` is `inputs_schema`.
For consistency, decide on one of the names, and use it in both places.
Preferably `schema_inputs`, as this name was used before your changes.